docs/topics/wasm/wasm-debugging.md

```diff
@@ -200,7 +201,7 @@ After setting up custom formatters, you can complete the debugging tutorial.
 
    ![Check call stack](wasm-debug-scope.png){width=550}
 
-   For an improved visualization of the variable values, see [Enhance your debugging with custom formatters](#enhance-your-debugging-with-custom-formatters).
+   For an improved visualization of the variable values, see [Use custom formatters](#use-custom-formatters).
```

Comment:
This link doesn't work when the "Use custom formatters" section is collapsed: the browser is trying to navigate to the anchor, but the content is not displayed. The link should probably lead to the "Configure your browser for debugging" heading while explaining that custom formatters are a part of this and the user needs to dig into it.